### PR TITLE
Add DEBUG=* debugging guide to readme

### PR DESCRIPTION
**Pull request by paulirish**
This technique was necessary for figuring out the odd behavior in #618

### CODE REVIEW

**Review by ebidel — APPROVED** [review]

**Comment by ebidel** [comment]
Just noticed we have https://github.com/GoogleChrome/puppeteer/blob/master/CONTRIBUTING.md#debugging-puppeteer too. Want to consolidate and move some of that over into the the readme debugging section? Doesn't make sense in contributing.

**Review by aslushnikov — APPROVED** [review]

**Comment by paulirish** [comment]
merged the contributing.md stuff in.

also cleaned up some whitespace (sorry)

File: README.md [hunk 1]
```diff
@@ -147,19 +147,35 @@ Explore the [API documentation](docs/api.md) and [examples](https://github.com/G
...
+
+   # Debug output can be enabled/disabled by namespace
+   env DEBUG="*,-*:protocol" node script.js # everything BUT protocol messages
+   env DEBUG="*:session" node script.js # protocol session messages (protocol messages to targets)
+   env DEBUG="*:mouse,*:keyboard" node script.js # only Mouse and Keyboard API calls
+
+   # Protocol traffic can be rather noisy. This example filters out all Network domain messages
+   env DEBUG="*" env DEBUG_COLORS=true node script.js 2>&1 | grep -v '"Network'
+   ```
+
```
Comment (ebidel) [inline README.md:175]: much lines brah

**Comment by ebidel** [comment]
LGTM

**Pull request merged**